algebra/src/main/scala/endpoints/EndpointAlg.scala

```diff
@@ -48,6 +48,24 @@ trait EndpointAlg extends UrlAlg with MethodsAlg {
     headers: RequestHeaders[C] = emptyHeaders
   )(implicit tuplerAB: Tupler.Aux[A, B, AB], tuplerABC: Tupler[AB, C]): Request[tuplerABC.Out]
 
+  /**
+    * Helper method to perform GET request
+    */
+  def get[A, C, AUnit](
+    url: Url[A],
+    headers: RequestHeaders[C] = emptyHeaders
+  )(implicit tuplerAB: Tupler.Aux[A, Unit, AUnit], tuplerABC: Tupler[AUnit, C]): Request[tuplerABC.Out] = request(Get, url, headers = headers)
```

Comment:
I thought compiler won't accept `Tupler[Tupler[A,Unit]#Out, C]#Out` as `Tupler[A,C]#Out` but know I see I was wrong and should've checked it. Thanks